app/Http/Controllers/CompetitionsController.php

```diff
@@ -7,7 +7,7 @@
 
 class CompetitionsController extends Controller
 {
-    private $validationRules = [
+    public static $validationRules = [
```

Comment:
It's really only the date assignment on the competition that matters for reasoning to have the form on the same page right?

Just feels very bad-practice-y to have the same form accessible from two different areas... maybe it should just default dates and then the admin can click on the competition to edit it directly and that'll take them to the `/competitions/:id/edit` page would be expected?